cmd/liqoctl/cmd/root.go

```diff
@@ -39,15 +41,20 @@ func NewRootCommand(ctx context.Context) *cobra.Command {
 	klogFlagset := flag.NewFlagSet("klog", flag.PanicOnError)
 	klog.InitFlags(klogFlagset)
 	klogFlagset.VisitAll(func(f *flag.Flag) {
-		if f.Name == "v" {
-			rootCmd.PersistentFlags().AddGoFlag(f)
+		if f.Name == "stderrthreshold" {
+			utilruntime.Must(f.Value.Set("FATAL"))
```

Comment:
Could be worth to add a comment and specify this is required to silence the helm library messages.